### ISSUE TITLE
Up max fps value to 65 on Android targets

### ISSUE THREAD

**Issue by jgranick** [openfl/openfl-native#182]
#182 Issue by __Hasufel__,

On weak targets, like OUYA, setting FPS at 65 compared to capping it at 60 (if we are still in the need of capping such a value) max helps a lot: a program set at 60fps will almost never hit the 60fps average and go for 55-57fps, whereas setting it at 65 will ensure hitting 60fps. One can enable.disable as well vsync to make sure everything is ok.

**Comment by jgranick** [comment]
Comment by __jgranick__:

We added this restriction, because in tests, it seemed that Android devices were limiting to a max rate of 60 FPS anyway, but if we had too high of a framerate, we could actually choke and starve the application so it would not run anymore.

We have new Android timing code, which appears to work well when the framerate is high. If I set a rate of 200 FPS here, it still runs smoothly (albeit at 60 FPS, but that's alright). We should keep an eye on it to make sure there are not threading/crash problems that occur when setting a high framerate, but for now it looks stable so I've removed the frame rate restriction entirely on Android.

Thanks!
